chore: update method signatures in line with ipfs/interface-ipfs-core#277

achingbrain opened the pull request:
See ipfs/interface-ipfs-core#277 for more information

vasco-santos commented on src/files/mv.js:
```diff
@@ -3,12 +3,24 @@
+    } else {
+      // support ipfs.file.mv(src, dest, opts, cb) and ipfs.file.cp(src1, src2, dest, opts, cb)
+      sources = args
     }
```
Maybe we can have the above logic abstracted in a util function since the code is replicated in `mv` and `cp`. What do you think @achingbrain ?

achingbrain commented:
@vasco-santos I've updated the PR and implemented your suggestion but I'm having problems getting Jenkins to pass.  Lots of instability around the ping tests.

They seem to be failing for master on Jenkins too, any ideas?

vasco-santos commented:
Thanks @achingbrain !

I checked the CI log and it was a timeout for windows. So, I started the CI again to check if it gets green again.

achingbrain commented:
> I checked the CI log and it was a timeout in windows environment. So, I started the CI again to check if it gets green again.

Ha, I've been restarting it all morning.  We should probably quarantine flaky tests until we can make them stable.

vasco-santos commented:
@achingbrain yeah! It still fails

vasco-santos commented:
Yeah, I agree that it is the best solution here @achingbrain !

achingbrain commented:
Great, can this be merged & released?

achingbrain commented:
...and could there please also be a corresponding release of [`ipfs/js-ipfsd-ctl`](https://github.com/ipfs/js-ipfsd-ctl) with the new `js-ipfs-api` version?

Releasing v1 of this modules would make this all so much easier!

vasco-santos commented:
For me, it is good to go and be released. 

I can handle the `ipfsd-ctl` part, but we need someone for releasing the API.

achingbrain commented:
I think everyone's on holiday..  I'll merge this and hopefully someone can do the do when people are back.
